lib/gcloud/dns/record/list.rb

```diff
@@ -45,6 +45,18 @@ def next
         end
 
         ##
+        # Retrieve all records and add to the current list.
+        # This changes the contained records.
```

Comment:
I think "This changes the contained records." sounds a little scary, and I don't think it's really accurate either. The contained records are unchanged, while any un-fetched records are added to them.